mgmt/rest/v2/config.go

```diff
@@ -159,3 +211,76 @@ func getPluginType(t string) (core.PluginType, error) {
 	}
 	return ityp, nil
 }
+
+// deletePluginConfigItemHelper builds different forms of request data into the way method deletePluginConfigItem accepts.
+// currently it accepts go-swagger client, swagger-ui and SanpCLI.
+func deletePluginConfigItemHelper(r *http.Request) ([]string, error) {
+	buf, err := ioutil.ReadAll(r.Body)
```

Comment:
@lmroz, r.Body will be used the downstream. This function is just a helper. I'll double check if all downstream closes it.